test/spec_lint.rb

```diff
@@ -225,6 +225,11 @@ def obj.fatal(*) end
     }.must_raise(Rack::Lint::LintError).
       message.must_match(/must start with/)
 
+    # XXX not actually sure what *should* happen in an OPTIONS * request.
+    lambda {
+      Rack::Lint.new(nil).call(env("REQUEST_METHOD" => "OPTIONS", "PATH_INFO" => ?*))
+    }.must_be_kind_of Proc
```

Comment:
`must_be_kind_of Proc` does not actually test anything.  You don't need to wrap this in a lambda, just execute the code and check the result.  You do have to pass a valid Rack app to `Rack::Lint.new`, though.

Reply:
Fair; it was late and I'm not familiar with minitest.

Reply:
New test is in place.